### DISCUSSION TITLE
v1.3.0-beta.3

### DISCUSSION THREAD

**Discussion by EvanMulawski** [EvanMulawski/FanControl.CorsairLink#62] [Announcements]
- New Device: Adds support for the AXi series of power supplies (Flex USB PSU).
  - Note: The minimum fan duty is 15%. When the fan power is set to 0%, control of the fan will be returned to the PSU allowing zero-RPM operation. When the fan power is set to 1% or higher, control of the fan will be returned to Fan Control.
  - Note: Communication with an AXi PSU requires driver installation. Instructions are provided in `Corsair-SiUsbXpress-Driver/README.txt`.
- General: Presents an alert if the incorrect build of Fan Control is detected.
- Logging: Presents an alert if multiple errors are detected. General logging improvements.

<hr /><em>This discussion was created from the release <a href='https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/v1.3.0-beta.3'>v1.3.0-beta.3</a>.</em>

**Comment by AssassinXDAhmed** [comment]
Hi there,
Is there any plan for "Lighting Node CORE" controller support?

**Reply by EvanMulawski** [reply]
Hi @AssassinXDAhmed, the Lighting Node Core is only a RGB LED controller - it does not control fan speeds.

**Reply by AssassinXDAhmed** [reply]
but I can't control the fan speeds.. I have 3 be quiet! fans and I can control them fine but the Corsair ones aren't controllable.. I exited iCUE and closed its services and still get the question mark sign when I try to control them.
The app reads the speed of the fans but it says its controlled by other software. I don't have any other software that control fan speeds other than App center by GIGABYTE and I already tried to close it but no luck.

**Reply by EvanMulawski** [reply]
What are the Corsair fans connected to? Feel free to post pictures and screenshots.

**Reply by AssassinXDAhmed** [reply]
One connected to System Fan 4 and the other to System Pump 6

**Reply by Fr0stX76** [reply]
> One connected to System Fan 4 and the other to System Pump 6

@AssassinXDAhmed. This is not a Corsair plugin issue, but rather a FanControl or LibreHardwareMonitor issue since it looks like something is not working between the fans and your motherboard. You should do some searching in the FanControl and LHM project issue sections and ask for support there if you cannot find the answer.

https://github.com/Rem0o/FanControl.Releases/issues
https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/issues

If that can help you, I suspect either the headers of your motherboard are not setup properly in FanControl, or they are not supported by the backend program (LHM).

Good luck!
